install/migrations/update_10.0.3_to_10.0.4/inventory.php

```diff
@@ -62,3 +62,13 @@
 
 //new right value for agent (previously based on config UPDATE)
 $migration->addRight('agent', READ | UPDATE | PURGE, ['config' => UPDATE]);
+
+//add new fields for Agent
+$migration->addField("glpi_agents", 'use_module_wake_on_lan', 'bool', ['value' => 0]);
+$migration->addField("glpi_agents", 'use_module_computer_inventory', 'bool', ['value' => 0]);
+$migration->addField("glpi_agents", 'use_module_esx_remote_inventory', 'bool', ['value' => 0]);
+$migration->addField("glpi_agents", 'use_module_remote_inventory', 'bool', ['value' => 0]);
+$migration->addField("glpi_agents", 'use_module_network_inventory', 'bool', ['value' => 0]);
+$migration->addField("glpi_agents", 'use_module_network_discovery', 'bool', ['value' => 0]);
+$migration->addField("glpi_agents", 'use_module_package_deployment', 'bool', ['value' => 0]);
+$migration->addField("glpi_agents", 'use_module_collect_data', 'bool', ['value' => 0]);
```

Comment:
`0` is already the default value.
```suggestion
$migration->addField("glpi_agents", 'use_module_wake_on_lan', 'bool');
$migration->addField("glpi_agents", 'use_module_computer_inventory', 'bool');
$migration->addField("glpi_agents", 'use_module_esx_remote_inventory', 'bool');
$migration->addField("glpi_agents", 'use_module_remote_inventory', 'bool');
$migration->addField("glpi_agents", 'use_module_network_inventory', 'bool');
$migration->addField("glpi_agents", 'use_module_network_discovery', 'bool');
$migration->addField("glpi_agents", 'use_module_package_deployment', 'bool');
$migration->addField("glpi_agents", 'use_module_collect_data', 'bool');
```